.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,113 @@
+    # Remaining workflows will be removed in the next scheduled run.
+    continue-on-error: true 
+    strategy:
+      fail-fast: false
+      matrix:
+        repo:
+          - 'jellyfin/jellyfin'
+          - 'jellyfin/jellyfin-web'
+          - 'jellyfin/jellyfin-androidtv'
+          - 'jellyfin/jellyfin-android'
+          - 'jellyfin/jellyfin-chromecast'
+          - 'jellyfin/jellyfin-sdk-csharp'
+          - 'jellyfin/jellyfin-sdk-kotlin'
+          - 'jellyfin/jellyfin-sdk-swift'
+          - 'jellyfin/jellyfin-client-axios'
+          - 'jellyfin/jellyfin-roku'
+          - 'jellyfin/jellyfin-vue'
+          - 'jellyfin/jellyfin-webos'
+          - 'jellyfin/jellyfin-plugin-anilist'
+          - 'jellyfin/jellyfin-plugin-anisearch'
+          - 'jellyfin/jellyfin-plugin-artwork'
+          - 'jellyfin/jellyfin-plugin-anidb'
+          - 'jellyfin/jellyfin-plugin-webhook'
+          - 'jellyfin/jellyfin-plugin-tvmaze'
+          - 'jellyfin/jellyfin-plugin-ldpauth'
+          - 'jellyfin/jellyfin-plugin-tvdb'
+          - 'jellyfin/jellyfin-plugin-fanart'
+          - 'jellyfin/jellyfin-plugin-tvheadend'
+          - 'jellyfin/jellyfin-plugin-bookshelf'
+          - 'jellyfin/jellyfin-plugin-opensubtitles'
+          - 'jellyfin/jellyfin-plugin-covertartarchive'
+          - 'jellyfin/jellyfin-plugin-kodisyncqueue'
+          - 'jellyfin/jellyfin-plugin-nextpvr'
+          - 'jellyfin/jellyfin-plugin-playbackreporting'
+          - 'jellyfin/jellyfin-plugin-musicbrainz'
+          - 'jellyfin/jellyfin-plugin-trakt'
+          - 'jellyfin/jellyfin-plugin-reports'
+          - 'jellyfin/jellyfin-plugin-autoorganize'
+          - 'jellyfin/jellyfin-plugin-tmdb'
+          - 'jellyfin/jellyfin-plugin-omdb'
+          - 'jellyfin/jellyfin-plugin-kitsu'
+          - 'jellyfin/jellyfin-plugin-tmdbboxsets'
+          - 'jellyfin/jellyfin-plugin-template'
+          - 'jellyfin/jellyfin-meta-plugins'
+          - 'jellyfin/jellyfin-media-player'
+          - 'jellyfin/jellyfin-mpv-shim'
+          - 'jellyfin/jellyfin-expo'
+          - 'jellyfin/jellyfin-meta'
+          - 'jellyfin/jellyfin-artwork'
+          - 'jellyfin/jellyfin.org'
+          - 'jellyfin/jellyfin-docs'
+          - 'jellyfin/jellyfin-kodi'
+          - 'jellyfin/jellyfin-metapackages'
+          - 'jellyfin/jellyfin-ffmpeg'
+          - 'jellyfin/Swiftfin'
+          - 'jellyfin/jellyfin-apiclient-javascript'
+          - 'jellyfin/jellyfin-exoplayer-ffmpeg-extension'
+          - 'jellyfin/Jellyfin.XmlTv'
+          - 'jellyfin/JavascriptSubtitlesOctopus'
+
+    steps:
+      - name: Clone repository
+      - uses: actions/checkout@v2.3.4
+
+      - name: Set correct permissions
+        run: chmod +x delete_workflows.sh
+        working-directory: ./maintenance
+
+      - name: Run deletion script 🗑
+        run: ./delete_workflows.sh ${{ matrix.repo }}
+        working-directory: ./maintenance
+        env:
+          GITHUB_TOKEN: ${{ secrets.JF_BOT_TOKEN }}
```

Comment:
Not sure how I should apply this. My script itself doesn't use the env var at all, it's the GitHub CLI (``gh``) the one responsible for the API queries. I configured it [as demoed in the GitHub docs](https://docs.github.com/en/actions/guides/using-github-cli-in-workflows)

Do you know if I can still mask it or gh will break with it?

Reply:
I realized that once I moved on, but yes you can still mask it. That is relay only for the log visible within a run where it will be replaced with `***` so no actual changes to the value of the env. And running this echo either within the action workflow yaml or in the external sh should be irrelevant, beside if you do it in the sh that a local execution would end up printing your token :S

Reply:
@h1dden-da3m0n Fixed!